requirements/base.txt

```diff
@@ -10,6 +10,7 @@ django-localflavor==4.0
 django-extensions==3.2.3
 djangorestframework==3.14.0
 django-redis==5.3.0
+django-middleware-global-request==0.3.5
```

Comment:
Lets assume we will merge https://github.com/DemocracyClub/WhoCanIVoteFor/pull/2169 ahead of this and I'll go back and change this